src/Posts.jsx

```diff
@@ -210,10 +192,7 @@ const loadMorePosts = () => {
     state.selectedTab == "following" ? "GetFollowingPosts" : "GetPostsQuery";
   const type = state.selectedTab;
 
-  if (
-    state.selectedTab == "following" &&
-    !state.accountsFollowing
-  ) {
+  if (state.selectedTab == "following" && !state.accountsFollowing) {
```

Comment:
Let's make sure we always use the more strict `===` over `==` when comparing values (lines 195 and 192).